src/plugins/data_view_management/public/components/edit_index_pattern/indexed_fields_table/indexed_fields_table.test.tsx

```diff
@@ -119,14 +122,15 @@ describe('IndexedFieldsTable', () => {
         schemaFieldTypeFilter={[]}
         fieldFilter=""
         compositeRuntimeFields={{}}
+        startServices={startServices}
         {...mockedServices}
       />
     );
 
     await new Promise((resolve) => process.nextTick(resolve));
     component.update();
 
-    expect(component).toMatchSnapshot();
+    expect({ items: component.props().children.props.items }).toMatchSnapshot();
```

Comment:
Needed to target a more specific field of data in the component tree, otherwise the snapshot would be huge.